tests/addresslib/parser_mailbox_test.py

```diff
@@ -559,10 +546,6 @@ def test_local_part():
         EmailAddress(sample_qpair_without_slashes, '"{0}"@b'.format(sample_qpair)))
 
     # fail quoted-pair
-    run_mailbox_test('"{0}""{0}"@b'.format(sample_qpair), None)
-    run_mailbox_test('"john""smith"@b'.format(sample_qpair), None)
```

Comment:
Valid address because we allow `Foo foo@bar.com` without `<>`. Equivalent to `"john" "smith"@b`.